src/PactNet.Native/Verifier/PactVerifier.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using PactNet.Verifier;
+
+namespace PactNet.Native.Verifier
+{
+    /// <summary>
+    /// Pact verifier
+    /// </summary>
+    public class PactVerifier : IPactVerifier
+    {
+        private readonly PactVerifierConfig config;
+        private readonly IDictionary<string, string> verifierArgs;
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="PactVerifier"/> class.
+        /// </summary>
+        public PactVerifier() : this(new PactVerifierConfig())
+        {
+        }
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="PactVerifier"/> class.
+        /// </summary>
+        /// <param name="config">Pact verifier config</param>
+        public PactVerifier(PactVerifierConfig config)
```

Comment:
There is a probleme here. The public constructor doesn't initialize the verifierArgs

Reply:
Ah that was a last minute refactor, doh